.github/workflows/prepare-release.yml

```
@@ -0,0 +1,23 @@
+name: Prepare Buildpack Releases
+
+on:
+  workflow_dispatch:
+    inputs:
+      bump:
+        description: "Bump"
+        required: true
+        default: 'patch'
```

(for consistency with the quotes used for the `description` field a few lines higher, plus matches the Prettier output style)

```suggestion
        default: "patch"
```